### PR TITLE
Unit Test -  Increase openssl's key size. Place test certs into a comon test folder

### PR DESCRIPTION
**Pull request by brbzull0**
Running autest in Fedora 33 with `OpenSSL 1.1.1i` seems to fail. Looking deeper it seems that the key zise policy increased in the newer openssl's.
This Draft contains a few changes:
1 - Regenerate basic certs (server.pem, server.key, signer.pem)
2 - Place the new keys in a common folder(`tests/tools/ssl`) and make relevant unit tests use from this new location.
3 - Update trafficserver.test.ext to export a new functions (`addDefaultSSLFiles` and  `addSSLFileFromDefaultTestFolder`) which can be used to add the files from the new common folder.
4 - Update microserver's certs .

### CODE REVIEW

**Comment by brbzull0** [comment]
[approve ci autest]

File: tests/gold_tests/timeout/tls_conn_timeout.test.py [hunk 1]
```diff
@@ -63,7 +63,7 @@
 # Request the port that should timeout in the handshake
 # Should retry once
 tr = Test.AddTestRun("tr-blocking-post")
-tr.Setup.Copy("../chunked_encoding/ssl/server.pem")
+tr.Setup.Copy("../../tools/ssl/server.pem")
```
Comment (bneradt) [inline tests/gold_tests/timeout/tls_conn_timeout.test.py:66]: Instead of ../../tools, I believe we can use the `Test.Variables.AtsTestToolsDir`.

**Review by bneradt — CHANGES_REQUESTED** [review]
Looks good. One small comment.

Thanks for updating the certs!

**Comment by shinrich** [comment]
Good cleanup and update for newer default openssl configs.  Once this lands, I'll do a review of my additional certs in the TLS area.

I think one of the comcast folks (@traeak ?) ran into this issue a few months ago.  A review from him would be good.

**Comment by randall** [comment]
This is a great PR! Thanks for the work here!

For future us, how does one regenerate the test certs?

**Comment by brbzull0** [comment]
> This is a great PR! Thanks for the work here!
> 
> For future us, how does one regenerate the test certs?

Hi @randall 
This [notes](https://github.com/yahoo/proxy-verifier/blob/ac9541ea7dec6a82ee7ea8b0e33249798a91b97e/test/keys/README) were given by @bneradt from Proxy Verifier.

Thanks.

**Comment by duke8253** [comment]
Confirmed this is working for Ubuntu as well, started seeing `ssl.SSLError: [SSL: EE_KEY_TOO_SMALL] ee key too small (_ssl.c:4021)` a few days ago.

**Review by randall — APPROVED** [review]

**Review by bneradt — APPROVED** [review]

**Comment by zwoop** [comment]
Cherry-picked to v9.1.x branch.

**Pull request merged**